packages/renderer/src/lib/preferences/item-formats/NumberItem.spec.ts

```diff
@@ -126,3 +130,25 @@ test('Expect decrement button only works one if minimum value is reached after o
 
   expect(input).toBeDisabled();
 });
+
+test('Expect zero value set correctly', async () => {
+  vi.useFakeTimers({ shouldAdvanceTime: true });
+  const record: IConfigurationPropertyRecordedSchema = {
+    id: 'record',
+    title: 'record',
+    parentId: 'parent.record',
+    description: 'record-description',
+    type: 'number',
+    minimum: -1,
+    maximum: 34,
+  };
+  const onChange = vi.fn();
+  const value = 1;
+  render(NumberItem, { record, value, onChange });
+
+  const input = screen.getByLabelText('decrement');
+  expect(input).toBeInTheDocument();
+  await userEvent.click(input);
+  vi.advanceTimersByTime(510);
```

Comment:
should it use timers or just ticks ?

Reply:
I think that we need to rely on timers, as the Svelte component is not refreshed, but only a call to onChange is called after 500ms:

```
    valueUpdateTimeout = setTimeout(() => {
      onChange(recordId, newValue);
      lastValue = newValue;
    }, 500);
```

Reply:
understood